app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,3 @@
+class UserMailer < ActionMailer::Base
+  default from: "barbaraklosowska@gmail.com"
```

Comment:
It's third repetition - please extract this to some setting to avoid duplications.